### PR TITLE
Promote AWS 2 Lambda to native

### PR DESCRIPTION
**Pull request by Jeansen**

### CODE REVIEW

**Review by oscerd — CHANGES_REQUESTED** [review]
I don't think this javascript should be in the extension.

**Comment by Jeansen** [comment]
> I don't think this javascript should be in the extension.

so, remove the whole lib folder with the .js files?

**Comment by Jeansen** [comment]
I see, the other extensions do not have it either, will remove the lib folder

**Review by ppalaga — APPROVED** [review]
Thanks @Jeansen ! A minor comment inline. Besides, if you could please use more descriptive commit messages next time. Issue title would be nice in addition to the issue number.

File: integration-tests/dropbox/pom.xml [hunk 1]
```diff
@@ -65,6 +65,8 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- The following dependencies guarantee that this module is built after them. You can update them by running `mvn process-resources -Pformat -N` from the source tree root directory -->
+
         <!-- The following dependencies guarantee that this module is built after them. You can update them by runing `mvn process-resources -Pformat -N` from the source tree root directory -->
```
[the remark this answers is not in the document]
Comment (Jeansen) [reply integration-tests/dropbox/pom.xml]: OK, I've removed the second line.

**Comment by Jeansen** [comment]
> Thanks @Jeansen ! A minor comment inline. Besides, if you could please use more descriptive commit messages next time. Issue title would be nice in addition to the issue number.

@ppalaga I am sorry, I've added the issue title to my commit.

**Comment by ppalaga** [comment]
It is perfect from my PoV now, thanks @Jeansen !

**Review by oscerd — APPROVED** [review]
Thanks a lot!

**Pull request merged**